joplin/templates/wagtailadmin/pages/create.html

```diff
@@ -12,27 +12,14 @@
     <input type="hidden" id="style_guide_url" name="style_guide_url" value="{{ style_guide_url }}">
     <div class="header secondary coa-header--secondary">
         <div>
-            <div class="create_page_header">
-              <div class="left col9 header-title">
-                  <h1 class="icon icon-doc-empty-inverse">
-                  {% trans 'New' %} <span>{{ content_type.model_class.get_verbose_name }}</span></h1>
-              </div>
-            </div>
+            <h1 class="create_page_header">
```

Comment:
There's a fair amount of sharing between the create and edit pages here. Can these header be turned into it's own template/partial that we include? We can pass in variables to turn off the share, preview and fancy draft buttons.

Reply:
@briaguya will this work? There will be a number of required fields empty? I think making all fields except title and topic optional isn't a great plan either. I'll add this comment in the cityofaustin/techstack#737 too

Reply:
Oh, I see, already noted in  cityofaustin/techstack#737 -- not sure I agree with the approach of only requiring on publish. It's not obvious to the content creator that a field is required until the last minute? I'll this note to the issue.